### PR TITLE
feat: change default font to Louis Goerge Cafe (#6)

### LINKED ISSUE #6: Use Louis George Café as default font

### PR DESCRIPTION
**Pull request by LordAldi**
Fixes #6

### CODE REVIEW

File: src/assets/scss/font.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@font-face {
+  font-family: "LouisGoergeCafe";
+  src: url("../font/Louis-George-Cafe-Italic.ttf");
+  font-weight: 400;
+  font-style: italic, oblique;
+}
+@font-face {
```
Comment (ariefrahmansyah) [inline src/assets/scss/font.scss:8]: Add line break here

File: src/assets/scss/font.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "LouisGoergeCafe";
+  src: url("../font/Louis-George-Cafe-Bold.ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+@font-face {
```
Comment (ariefrahmansyah) [inline src/assets/scss/font.scss:22]: Add line break here

File: src/assets/scss/font.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  font-family: "LouisGoergeCafe";
+  src: url("../font/Louis-George-Cafe.ttf");
+  font-weight: 400;
+  font-style: normal;
+}
+*,
```
Comment (ariefrahmansyah) [inline src/assets/scss/font.scss:29]: Add line break here.

**Review by ariefrahmansyah — CHANGES_REQUESTED** [review]
Please put the font files in `assets/fonts/Louis-George-Cafe` for better organization.

**Review by ariefrahmansyah — APPROVED** [review]
LGTM

**Review by fadelarmando — APPROVED** [review]
LGTM

**Pull request merged**